Kitodo/src/main/java/de/sub/goobi/config/Parameters.java

```diff
@@ -0,0 +1,695 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package de.sub.goobi.config;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * These constants define configuration parameters usable in the configuration
+ * file. This file reflects the order of the global
+ * {@code kitodo_config.properties} file (as far as possible), and vice versa.
+ */
+public class Parameters {
+
+    // =============================================================================
+    // FILE AND DIRECTORY MANAGEMENT
+    // =============================================================================
```

Comment:
Such a lines in comments are not helping to read it.

Reply:
This is taken from `kitodo_config.properties` to make both files look alike.

![equal](https://user-images.githubusercontent.com/3040657/42273460-7ffac57c-7f89-11e8-876c-17caa4d80729.PNG)

We can change it, but then we should change `kitodo_config.properties` accordingly as well.